lectures/eigen.md

```diff
@@ -502,13 +577,20 @@ It is quite evident that the transformation $AB$ is not the same as the transfor
 
 ## Iterating on a Fixed Map
 
-In economics we often are interested in analyzing the behavior of iterating on a fixed map.
+In economics (and especially in dynamic modeling), we often are interested in
+analyzing behavior where we repeatedly apply a fixed matrix.
 
-Consider a vector $v$ under the mapping $A$. 
+For example, given a vector $v$ and a matrix $A$, we are interested in
+studying the sequence
 
-We are interested in what happens when we repeatedly multiply $v$ by $A$, i.e., what happens to $AAv = A^2v$, $A^3v$ and so on for $A^kv$
+$$ 
+    v, \quad
+    Av, \quad
+    AAv = A^2v, \ldots
+$$
 
-Let's first see examples of a sequence of iterates $(A^kx)_{k \geq 0}$ under different maps $A$.
+Let's first see examples of a sequence of iterates $(A^k x)_{k \geq 0}$ under
+different maps $A$.
```

Comment:
The ``matrix_power`` in the following python code cell needs to be defined or imported.

Useful resources:
- https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.linalg.matrix_power.html
- https://numpy.org/doc/stable/reference/generated/numpy.linalg.matrix_power.html